src/modinfo.h

```diff
@@ -58,6 +58,7 @@ struct sr_mod_info_s {
         uint32_t xpath_count;   /**< Count of XPaths. */
         uint32_t state;         /**< Module state (flags). */
         uint32_t request_id;    /**< Request ID of the published event. */
+        uint32_t reuse_diff;    /**< Whether a reusable diff has been written into the shm for this request_id. */
```

Comment:
Is this the flag you want me to change to `shm_data_diff_req_id`?

Reply:
Yes, that is what I was referring to.

Reply:
I don't think it will be automatically outdated for `sr_shmsub_rpc_internal_call_callback`, which is the only function for which resetting this flag is required.

Reply:
Yes, I mistakenly thought the flag is global.

Reply:
It was not clear from the previous commit message, but I have added some more explanation to the commit now. Hopefully it is adequate now.